ggshield/core/config/user_config.py

```diff
@@ -45,10 +45,8 @@ class SecretConfig(FilteredConfig):
     ignored_matches: List[IgnoredMatch] = field(default_factory=list)
     ignored_paths: Set[str] = field(default_factory=set)
     ignore_known_secrets: bool = False
-    prereceive_remediation_message: str = """A pre-receive hook set server side prevented you from pushing secrets.
-Since the secret was detected during the push BUT after the commit, you need to:
-1. rewrite the git history making sure to replace the secret with its reference (e.g. environment variable).
-2. push again."""
+    # if configuration key left unset the client remediation are used.
```

Comment:
Nit: the comment is not super clear. "Client" may be a bit ambiguous here. I would say the "dashboard's remediation message is used". Feel free to ignore this one.